### ISSUE TITLE
Save Scenario to .csv file

### ISSUE THREAD

**Issue by mostaphaRoudsari** [mostaphaRoudsari/EmergeSettlement#23]
I'm pretty sure it will come up at some point to be able to save and load the whole scenario (all the slider values and results. I put it here as a wish for now.

**Comment by brahamw** [comment]
yes, as I was working on it today, the values kept changing to whoever was testing a scenario... I entered the values I was testing in the column next to the parameter settings in sheet4
